forge-gui/res/cardsfolder/upcoming/demolition_field.txt

```diff
@@ -0,0 +1,10 @@
+Name:Demolition Field
+ManaCost:no cost
+Types:Land
+A:AB$ Mana | Cost$ T | Produced$ C | SpellDescription$ Add {C}.
+A:AB$ Destroy | Cost$ 2 T Sac<1/CARDNAME> | ValidTgts$ Land.nonBasic+OppCtrl | TgtPrompt$ Select target nonbasic land an opponent controls | SubAbility$ DBSearch | AILogic$ GhostQuarter | SpellDescription$ Destroy target nonbasic land an opponent controls. That land's controller may search their library for a basic land card, put it onto the battlefield, then shuffle. You may search your library for a basic land card, put it onto the battlefield, then shuffle.
```

Comment:
How does the Stack Description look when you activate this ability?

Reply:
![image](https://user-images.githubusercontent.com/67333662/200600959-2a5f2135-bdc7-4807-9d9b-5c258bdc21c9.png)

Just spelldescription